webpack.common.js

```diff
@@ -8,8 +8,13 @@ module.exports = {
         middleware: './src/scripts/pages/login/middleware.js',
         zingchat: './src/scripts/pages/zingchat/main.js',
```

Comment:
đổi tên lại e, sai chính tả, zingchart not zingchat
em đổi tên file luôn, nói chung check all source những chỗ nào đag sai chính tả thì đổi lại